django_db_geventpool/backends/postgresql_psycopg2/base.py

```diff
@@ -14,144 +14,29 @@
 except ImportError:
     from eventlet.semaphore import Semaphore
 
-try:
-    from django.db.backends.postgresql_psycopg2.base import CursorWrapper
-except ImportError:
-    pass
+from django.db.backends.postgresql.base import DatabaseWrapper as OriginalDatabaseWrapper
 
-import django
-from django.db.backends.postgresql_psycopg2.base import \
-    DatabaseWrapper as OriginalDatabaseWrapper
-from django.db.backends.signals import connection_created
-from django.conf import settings
-from django.db.backends.postgresql_psycopg2.base import utc_tzinfo_factory
-from django.utils.encoding import force_str
-
-try:
-    import psycopg2_pool as psypool
```

Comment:
I can't tell what this was for. Is/was there a package called `psycopg2_pool`? Or was this just a backwards way to import `django_db_geventpool/backends/postgresql_psycopg2/psycopg2_pool.py`?

Reply:
Sorry for the delay in the response, I can't remember now, but it seems to be something for backwards compatibility, anyway, I think that changing it to a relative import should be ok